vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Provide usage documentation for your packages
+description: Guidance for adding usage documentation to packages
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 07/10/2023
+---
+# Provide usage documentation for your packages
```

Comment:
```suggestion
# Provide usage documentation for your ports
```

Reply:
We should discuss which of these terms we want to use in the title